packages/playground/src/components/qr_play_store.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="d-flex flex-column text-center align-center">
-    <slot></slot>
+    <p><slot></slot></p>
```

Comment:
Why are we wrapping the slot itself? we should put the tag element on the slot content right ?

Reply:
it's the same